server.js

```diff
@@ -6287,6 +6287,92 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// MicroBadger integration.
+camp.route(/^\/microbadger\/(image-size|layers)\/([^/]+)\/([^/]+)\/?([^/]*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var type = match[1];
+  var user = match[2];
+  var repo = match[3];
+  var tag = match[4];
+  var format = match[5];
+  if (user === '_') {
+    user = 'library';
+  }
+  var path = user + '/' + repo;
+  var url = 'https://api.microbadger.com/v1/images/' + path;
```

Comment:
I know a lot of the code in `server.js` uses `var`, but when adding new code maybe it would be better to make use of `let` and `const` where possible. For example `url`, `repo`, `format` etc here are declared and never re-assigned so could be declared with `const`. Similarly, `i`, `j`, `versionJson`, `tagJson` further down are unnecessarily defined with function scope and could be block-scoped by declaring them with `let`.

Reply:
Yep, I'll change it. Totally makes sense. I just see that the most part uses `var` and the linter doesn't complain, so, I tried to be like majority :)